### PR TITLE
Adding a hint that changing local env vars requires restarting `yarn …

### PR DESCRIPTION
**Pull request by ryancwalsh**
…rw dev`

It took me far too long to realize this.

### CODE REVIEW

File: docs/environmentVariables.md [hunk 1]
```diff
@@ -150,6 +150,7 @@ config({
   defaults: path.join(getPaths().base, '.env.defaults'),
 })
 ```
+Remember, if `yarn rw dev` is already running, your local app won't reflect any changes you make to your `.env` until you stop and re-run `yarn rw dev`.
```
Comment (cannikin) [inline docs/environmentVariables.md:153]: This is going to seem like the most petty request ever, but could you add a newline above this, between this paragraph and the end of the code block above? Sorry! Other than that, love it! :)

**Review by cannikin — CHANGES_REQUESTED** [review]

File: docs/environmentVariables.md (hunk 1 above)
Comment (ryancwalsh) [reply docs/environmentVariables.md:154]: No problem at all. Hopefully I understood you correctly: https://github.com/redwoodjs/redwoodjs.com/pull/422/commits/2ad8a767f3dd3e120cb0d4fb44cb0a5f10323a37

File: docs/environmentVariables.md [hunk 2]
```diff
@@ -151,6 +151,8 @@ config({
 })
 ```
 
+Remember, if `yarn rw dev` is already running, your local app won't reflect any changes you make to your `.env` until you stop and re-run `yarn rw dev`.
```
Comment (Tobbe) [inline docs/environmentVariables.md:154]:
```suggestion
Remember, if `yarn rw dev` is already running, your local app won't reflect any changes you make to your `.env` file until you stop and re-run `yarn rw dev`.
```

**Comment by Tobbe** [comment]
@ryancwalsh Thanks for your changes. I'm sure it will save others some frustration when messing around with the env vars 🙂

**Pull request merged**